code/modules/mob/living/silicon/robot/dogborg/dog_sleeper_vr.dm

```diff
@@ -448,6 +447,9 @@
 					items_preserved |= T
 				else
 					hound.cell.charge += (50 * digested)
+			if(istype(target,/obj/effect/decal/remains))
```

Comment:
Should this be else if? I think the else below might try and preserve already-digested items as-is.